randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -446,21 +461,31 @@ Extra - asset_id: 13
 > Door to Terminator Room; Heals? False
   * Layers: default
   * Normal Door to Terminator Room/Door to Green Pirates Shaft
-  > Door to Lower Mushrooms
-      Trivial
   > Door to Gold Four Hallway
-      Trivial
+      Any of the following:
+          Charge Beam or Ice Beam or Missile or Plasma Beam or Power Bombs or Spazer Beam or Super Missile or Wave Beam or Can Lay Any Bombs
+          # 3 hits for Intermediate damage boosting
+          Damage Boosting (Intermediate) and Normal Damage ≥ 61
+          # 5 hits for beginner damage boosting
+          Damage Boosting (Beginner) and Normal Damage ≥ 101
 
 > Door to Lower Mushrooms; Heals? False
   * Layers: default
   * Normal Door to Lower Mushrooms/Door to Green Pirates Shaft
-  > Door to Terminator Room
+  > Door to Gold Four Hallway
       Trivial
 
 > Door to Gold Four Hallway; Heals? False
   * Layers: default
   * Missile Door to Gold Four Hallway/Door to Green Pirates Shaft
   > Door to Terminator Room
+      Any of the following:
+          Charge Beam or Ice Beam or Missile or Plasma Beam or Power Bombs or Spazer Beam or Super Missile or Wave Beam or Can Lay Any Bombs
```

Comment:
This is still missing a comment. soemthign like "defeat pirate" or sth

Reply:
Done.